feat(crossfade) :beers: :sparkles: add beeper crossfade

ericheep opened the pull request:
A beeper will get placed into an array of beepers
upon its creation, with the remaining beepers fading out
until they are stopped.

:beers::beers:

ericheep committed:
(feat) :beers: :sparkles: Add Beeper Crossfade

A beeper will get placed into an array of beepers
upon its creation, with the remaining beepers fading out
until they are stopped.

clarkenciel commented on src/index.js:
```diff
@@ -161,12 +161,20 @@ var audioCtx = new window.AudioContext({
 
-const createNewBeeper = () => {
+// array to hold our dying beepers until they gracefully fade away
+var beepers = []
```
at this point we may as well store this in the store

ericheep replied:
You're right, I'll work on a commit for the store.

ericheep committed:
(fix) Band-aid pop

Must be a better way to do this, but pop
works pretty damn well for now :beers:.

clarkenciel commented:
hm also, if we're gonna go with gitmoji style this should be:
🍺 ✨ (index.js): add beeper crossfade

fakemonster committed:
:bug: :recycle: (beepers) keep sound smooth after much drawing

Sound would get weird when drawing many circles on canvas

Moved beepers into store and made cleanup asynchronous
